test/jdk/java/util/regex/NegativeArraySize.java

```diff
@@ -26,17 +26,17 @@
  * @bug 8223174
  * @summary Pattern.compile() can throw confusing NegativeArraySizeException
  * @requires os.maxMemory >= 5g
- * @run main/othervm NegativeArraySize -Xms5G -Xmx5G
+ * @run testng/othervm -Xms5G -Xmx5G NegativeArraySize
```

Comment:
I note that the order of the arguments has changed. Will that work as expected? Had it worked as expected before?

Reply:
The new order is consistent with other tests. I had difficulty getting it to run in the original configuration. Perhaps jtreg is more sensitive on order with the testng runner.